tests/Model/ModelManagerTest.php

```diff
@@ -742,8 +751,15 @@ public function testModelReverseTransform(): void
         $this->assertSame('hello', $object->getMultiWordProperty());
     }
 
+    /**
+     * NEXT_MAJOR: Remove this test.
+     *
+     * @group legacy
+     */
     public function testCollections(): void
     {
+        $this->expectDeprecation('Method Sonata\DoctrineORMAdminBundle\Model\ModelManager::getModelCollectionInstance() is deprecated since sonata-project/doctrine-orm-admin-bundle 3.x and will be removed in version 4.0.');
+
```

Comment:
In the other tests, there is no extra LF between the expectation and the method call.

Reply:
Done